azure-pipelines.yml

```diff
@@ -12,13 +12,13 @@ jobs:
       matrix:
         Python36:
           python.version: "3.6"
-          tensorflow.version: "1.13.1"
+          tensorflow: "tensorflow==1.13.1"
         Python37:
           python.version: "3.7"
-          tensorflow.version: "1.13.1"
+          tensorflow: "tensorflow==1.13.1"
         Python37TF2:
           python.version: "3.7"
-          tensorflow.version: "2.0.0-alpha0"
+          tensorflow: "tf-nightly-2.0-preview"
```

Comment:
What exactly is affected by this? I would prefer not having to use the nightly build.

Reply:
This is our CI on Azure Pipelines that executes the tests for TF 2.

`2.0.0-alpha0` has a bug that prevents the `QuantizationLogger` from working properly. This is fixed in the latest nightly. I think in any case it makes sense to test against the nightly build, so we can see when things break